chall07/vsempere.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"strconv"
+	"regexp"
+	"log"
+	"io/ioutil"
+	"net/http"
+)
+
+func toHex(dec []byte) string {
+	ri, err := strconv.Atoi(string(dec))
+	if err != nil {
+		log.Fatal(err)
+	}
+	str := strconv.FormatInt(int64(ri), 16)
+	if len(str) < 2 {
+		return "0"+ str
+	} else {
+			return str
+	}
```

Comment:
There is a much simpler way to do this, you could use `fmt.Sprintf` with flags:
```suggestion
	return fmt.Sprintf("%02x", ri)
```